library/modules/bigip_gtm_dns_listener.py

```diff
@@ -0,0 +1,836 @@
+        When C(absent), ensures that the pool is removed from the system. When
+        C(enabled) or C(disabled), ensures that the pool is enabled or disabled
+        (respectively) on the remote device.
+    type: str
+    choices:
+      - present
+      - absent
+      - enabled
+      - disabled
+extends_documentation_fragment: f5networks.f5_modules.f5
+author:
+  - Andrey Kashcheev (@andreykashcheev)
+'''
+
+EXAMPLES = r'''
+- name: Create a DNS Listener
+  bigip_gtm_dns_listener:
+    name: test-dns-listener
+    description: this is a test DNS listener
+    address: 192.0.1.0
+    port: 30025
+    advertise: no
+    ip_protocol: tcp
+    translate_address: enabled
+    translate_port: enabled
+    auto_lasthop: default
+    source_port: preserve
+    vlan_enabled: yes
+    vlans:
+      - /Common/my-vlan
+    mask: 255.255.255.0
+    provider:
+      password: secret
+      server: lb.mydomain.com
+      user: admin
+  delegate_to: localhost
+'''
+
+RETURN = r'''
+name:
+  description: DNS Listener name.
+  returned: changed
+  type: str
+  sample: test-dns-listener
+mask:
+  description: subnet mask used by listener to identify address range.
+  returned: changed
+  type: str
+  sample: 255.255.0.0
+address:
+  description: IP address on which the system listens.
+  returned: changed
+  type: str
+  sample: 10.0.0.2
+port:
+  description: port on which the system listens.
+  returned: changed
+  type: int
+  sample: 53
+source_port:
+  description: specifies if system preservs the source port of the connection.
+  returned: changed
+  type: str
+  sample: preserve
+advertise:
+  description: specifies if listener advertises to surrounding routers.
+  returned: changed
+  type: bool
+  sample: yes
+auto_lasthop:
+  description: shows if automatically map last hop for pools.
+  returned: changed
+  type: str
+  sample: default
+translate_address:
+  description: specifies if address translation is enabled.
+  returned: changed
+  type: str
+  sample: enabled
+translate_port:
+  description: specifies if port translation is enabled.
+  returned: changed
+  type: str
+  sample: enabled
+fallback_persistence:
+  description: fallback persistence profile for the listener to use when the default persistence profile is not available.
+  returned: changed
+  type: str
+  sample: /Common/fallback-profile
+enabled:
+  description: provides DNS listener state.
+  returned: changed
+  type: bool
+  sample: yes
+ip_protocol:
+  description: ip protocol used by DNS listener.
+  returned: changed
+  type: string
+  sample: tcp
+vlan_enabled:
+  description: indicates if traffic will be accepted from specified vlans.
+  returned: changed
+  type: bool
+  sample: no
+vlans:
+  description: list of vlans used by DNS Listener
+  returned: changed
+  type: list
+  sample: ['/Common/vlan1', '/Common/vlan2']
+rules:
+  description: list of rules run by DNS Listener
+  returned: changed
+  type: list
+  sample: ['/Common/rule1', '/Common/rule2']
+'''
```

Comment:
capital letters and full stops

Reply:
Updated all to upper-case and add missing fullstop. 

This comment is addressed now